### PR TITLE
Alpha fr#712

### LINKED ISSUE #764: [Other Issues]: `Development-Workflow.md` needs to state what the builds are (i.e. nightly)
The above page is linked to the text "Cadence" which indicates / infers a release cycle. but it just mentions branches ??

image

### PR DESCRIPTION
**Pull request by Wagnerp**
* Implemented #712 
* Implemented #761 
* Resolved #764 

image

### CODE REVIEW

**Comment by Smurf-IV** [comment]
Comments left in https://github.com/Krypton-Suite/Standard-Toolkit/issues/764

File: Documents/Help/Changelog.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 =======
 
 ## 2022-11-xx - Build 2211 - November 2022 <!--Possible August or September release?-->
+* Implemented [#761](https://github.com/Krypton-Suite/Standard-Toolkit/issues/761), Can TFM also include .NET Framework 4.8.1? (**Note:** This only applies to users of Visual Studio 2022)
```
Comment (Smurf-IV) [inline Documents/Help/Changelog.md:9]: Should state "Currently" as it will become a public "full redist" (As that is what is used by vs 2022 anyway)

Comment (Wagnerp) [reply Documents/Help/Changelog.md]: Resolved

**Comment by Wagnerp** [comment]
@Smurf-IV I'm going to put in your changes that were made yesterday into this branch, as this PR may have wiped them out.

**Pull request merged**